includes/class-llms-loader.php

```diff
@@ -18,6 +18,65 @@
  */
 class LLMS_Loader {
 
+	/**
+	 * These classes do not conform to any of the LifterLMS class name or file name standards.
+	 *
+	 * @since [version]
+	 *
+	 * @var string[] [ lowercase_class_name => path_relative_to_LLMS_PLUGIN_DIR ]
+	 */
+	private $non_standard_classes = array(
+		// Missing "_Abstract_" from class name.
+		'llms_admin_metabox'            => 'includes/abstracts/abstract.llms.admin.metabox.php',
+		'llms_admin_table'              => 'includes/abstracts/abstract.llms.admin.table.php',
+		'llms_analytics_widget'         => 'includes/abstracts/abstract.llms.analytics.widget.php',
```

Comment:
There are dozens of include statements that can be removed with this change to the autoloader. I wasn't sure if you wanted me to go down that far into the rabbit hole, or wait until later?

Reply:
I didn't want you to go into this rabbit hole *at all* but since we're already here and I've noticed these things then yes, we should remove them. Perhaps as a separate PR so we don't pollute this one further. Or open this as a new issue to track and clean up in the future.

Reply:
I have removed loading of all class files that don't instantiate their class in favor of autoloading.